src/Organizer/OrganizerLDProjector.php

```diff
@@ -366,6 +374,39 @@ private function newDocument($id)
     }
 
     /**
+     * @param TitleUpdated $titleUpdated
+     * @param Language|null $language
+     */
+    private function applyTitle(
+        TitleUpdated $titleUpdated,
```

Comment:
Is dus 'automatisch' mee aangepast door TitleTranslated laten af te leiden van OrganizerEvent en de signatuur van applyTitle aan te passen.